src/Filesystem/Folder.php

```diff
@@ -194,16 +194,32 @@ public function read($sort = true, $exceptions = false, $fullPath = false)
             if ($fullPath) {
                 $name = $item->getPathname();
             }
+
             if ($item->isDir()) {
-                $dirs[] = $name;
+                $dirs[$item->getCTime()][] = $name;
             } else {
-                $files[] = $name;
+                $files[$item->getCTime()][] = $name;
```

Comment:
No problem mate. Thanks for clearing it to me.
Actually I am having the correct structure (I attach a photo). I guess you have the last patch, right?

![image](https://cloud.githubusercontent.com/assets/5722357/14935860/fbb8e7f0-0ee0-11e6-89dd-1d035fc8dd60.png)

The two errors I am getting in travis shows that the result have different order, quite surprising because in my local test all are correct. Maybe @markstory could point me in the right way about why this.

Thanks both of you